### DISCUSSION TITLE
Redash pricing plan

### DISCUSSION THREAD

**Discussion by rdorottya** [getredash/redash#5920] [General]
Hi all,

There is only one version of Redash (open source) or there is another version with subscription? If there is one with subcription, what is the main difference between the two version?

I am searching for a BI tool that contains the following features:
- real-time reporting (maximum update time of 10 seconds)
- table visualisation
- charts, trends
- drill down, drill through
- embedded analytics (wallboard feature - users can add or remove premade graphs from dashboards)
- REST-API
- PostgreSQL connection
- row level security
- on premise

Does Redash include the features mentioned above?

**Comment by bun4uk** [comment]
Redash is great open source BI system. We're using it for 3 years. But it almost no new development after Databricks bought it. 
And they shout down cloud version of Redash. Only self hosted version is available.

**Reply by sdoppani** [reply]
Is there a whitelabel solution available for Redash?

**Reply by bun4uk** [reply]
> Is there a whitelabel solution available for Redash?

what do you mean by "whitelabel" solution?

**Reply by Jengro777** [reply]
refer to：#6033
